benchmark/README.md

```diff
@@ -0,0 +1,19 @@
+# Benchmarks
+
+1) The following graphs show the benchmark results (runtime (in secs)) of running match attempts, using different PSI algorithms, on various Google Cloud VMs [general-purpose virtual machines (VMs)](https://cloud.google.com/compute/docs/general-purpose-machines#n2-standard) like n2-standard-32, n2-standard-48, n2-standard-64 and n2-standard-80. For each experiment, same type of VMs are used for both the sender and the receiver. The graphs are generated by performing match attempts between a receiver with a fixed dataset of 50m records and a sender with datasets of 50m, 100m, 150m, 200m and 250m records. The bpsi used for these experiments has a fixed false positive rate of 1e-6.
```

Comment:
1) The following graphs show the results of benchmarking match attempts using different PSI algorithms on various Google Cloud VM's [general-purpose virtual machines (VMs)](https://cloud.google.com/compute/docs/general-purpose-machines#n2-standard) like n2-standard-64. In each benchmark, the sender and receiver use the same model VM. The receiver has 50m records while the sender has varying datasets of 50m, 100m, 150m, 200m and 250m records. The bpsi used for these experiments has a false positive rate fixed at 1e-6.